src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -108,7 +108,34 @@ public enum ListType {
                                      false, true),
     RPT_SOUTHERN_WEEKLY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
         RPT_LONDON_WEEKLY_HEARING_LIST, "Residential Property Tribunal: Southern Region Weekly Hearing List",
-                                     false, true);
+                                     false, true),
+    SSCS_MIDLANDS_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        null, "Midlands First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+                                   false, true),
+    SSCS_SOUTH_EAST_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
+        SSCS_MIDLANDS_DAILY_HEARING_LIST,
+        "South East First-tier Tribunal (Social Security and Child Support) Daily Hearing List",
+    false, true),
+    SSCS_WALES_AND_SOUTH_EAST_DAILY_HEARING_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES,
```

Comment:
I think there is a mistake here. The list type before is south east. This one should be Wales and South West (not East). Please check with Ogechi and update accordingly.

Reply:
Oegechi confirmed that it should be Wales South West. it's fixed.